Makefile

```diff
@@ -26,7 +26,7 @@ local_generate:: bin/pulumi-java-gen
 generate_schema:: bin/pulumi-java-gen
 	echo "Generating Pulumi schema..."
 	$(WORKING_DIR)/bin/$(CODEGEN) schema ${VERSION}
-	$(WORKING_DIR)/bin/$(JAVA_GEN) generate --schema $(WORKING_DIR)/provider/cmd/$(PROVIDER)/schema.json --out sdk/java --build gradle-nexus
+	#$(WORKING_DIR)/bin/$(JAVA_GEN) generate --schema $(WORKING_DIR)/provider/cmd/$(PROVIDER)/schema.json --out sdk/java --build gradle-nexus
```

Comment:
Still needed? Actually, it looks like the whole Makefile should be updated from `ci-mgmt`.

Reply:
Doh - this leaked through from my dev iteration changes. Will remove this.